addons/grenades/CfgAmmo.hpp

```diff
@@ -97,4 +108,23 @@ class CfgAmmo {
         effectsSmoke = "ACE_M84FlashbangEffect";
         whistleDist = 0;
     };
+    
+    class Chemlight_base: SmokeShell {
+        soundImpactHard1[] = {"A3\sounds_f\characters\footsteps\concrete_run_1",1,1.8,65};
+        soundImpactHard2[] = {"A3\sounds_f\characters\footsteps\concrete_run_1",1,1.9,65};
+        soundImpactHard3[] = {"A3\sounds_f\characters\footsteps\concrete_run_1",1,1.6,65};
+        soundImpactHard4[] = {"A3\sounds_f\characters\footsteps\concrete_run_1",1,2,65};
+        soundImpactHard5[] = {"A3\sounds_f\characters\footsteps\concrete_run_1",1,1.5,65};
+        soundImpactHard6[] = {"A3\sounds_f\characters\footsteps\concrete_run_1",1,1.7,65};
+        soundImpactHard7[] = {"A3\sounds_f\characters\footsteps\concrete_run_1",1,2.1,65};
+        soundImpactIron1[] = {"A3\sounds_f\characters\footsteps\concrete_run_1",1,2.1,95};
+        soundImpactIron2[] = {"A3\sounds_f\characters\footsteps\steel_run_4",1,1.75,95};
+        soundImpactIron3[] = {"A3\sounds_f\characters\footsteps\steel_run_4",1,1.6,95};
+        soundImpactIron4[] = {"A3\sounds_f\characters\footsteps\steel_run_2",1,2,95};
+        soundImpactIron5[] = {"A3\sounds_f\characters\footsteps\steel_run_2",1,2.1,95};
+        soundImpactWoodExt1[] = {"A3\sounds_f\characters\footsteps\wood_run_ext_1",1,1.1,75};
+        soundImpactWoodExt2[] = {"A3\sounds_f\characters\footsteps\wood_run_ext_1",1,1.15,75};
+        soundImpactWoodExt3[] = {"A3\sounds_f\characters\footsteps\wood_run_ext_2",1,1.1,75};
+        soundImpactWoodExt4[] = {"A3\sounds_f\characters\footsteps\wood_run_ext_2",1,1.15,75};
```

Comment:
Why are there dupicates here? You can adjust the array belonging to these instead:
```
        impactGroundSoft[] = {"soundImpactSoft1",0.2,"soundImpactSoft2",0.2,"soundImpactSoft3",0.2,"soundImpactSoft4",0.1,"soundImpactSoft5",0.1,"soundImpactSoft6",0.1,"soundImpactSoft7",0.1};
        impactGroundHard[] = {"soundImpactHard1",0.2,"soundImpactHard2",0.2,"soundImpactHard3",0.2,"soundImpactHard4",0.1,"soundImpactHard5",0.1,"soundImpactHard6",0.1,"soundImpactHard7",0.1};
        impactIron[] = {"soundImpactIron1",0.2,"soundImpactIron2",0.2,"soundImpactIron3",0.2,"soundImpactIron4",0.2,"soundImpactIron5",0.2};
        impactArmor[] = {"soundImpactIron1",0.2,"soundImpactIron2",0.2,"soundImpactIron3",0.2,"soundImpactIron4",0.2,"soundImpactIron5",0.2};
        impactBuilding[] = {"soundImpactHard1",0.2,"soundImpactHard2",0.2,"soundImpactHard3",0.2,"soundImpactHard4",0.1,"soundImpactHard5",0.1,"soundImpactHard6",0.1,"soundImpactHard7",0.1};
        impactFoliage[] = {"soundImpactSoft1",0.2,"soundImpactSoft2",0.2,"soundImpactSoft3",0.2,"soundImpactSoft4",0.1,"soundImpactSoft5",0.1,"soundImpactSoft6",0.1,"soundImpactSoft7",0.1};
        impactWood[] = {"soundImpactWoodExt1",0.25,"soundImpactWoodExt2",0.25,"soundImpactWoodExt3",0.25,"soundImpactWoodExt4",0.25};
```

etc.